### DISCUSSION TITLE
fetcher.load vs useQuery for client-side API calls?

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [remix-run/remix#4090] [Q&A]
> For folks who are coming from `react-query` or a similar hooks data fetching library & are now using Remix:

### Question
1. Is it safe to say that [`fetcher.load`](https://remix.run/docs/en/v1/api/remix#usefetcher) could be swapped with react-query's  [useQuery hook](https://tanstack.com/query/v4/docs/reference/useQuery?from=reactQueryV3&original=https://react-query-v3.tanstack.com/reference/useQuery) to fetch data not associated with UI routes (popovers, dynamic forms, etc.)? 
2. What am I missing out on if I don't use `useFetcher` when using fullstack remix?
   - reason I ask this is because I know remix does other cool stuff like revalidate a page if your making a mutation to a UI route
   

I want to learn to use Remix to the best of my ability
maybe I'm not using  `useFetcher` in the best way or not seeing things right :sweat_smile:
   
### Related
- https://github.com/remix-run/remix/discussions/4089
- [Remix Discord thread](https://discord.com/channels/770287896669978684/770287896669978687/1013273394520789124)

**Comment by refactornator** [comment]
Did you end up trying `useQuery` instead of fetchers, and did it work out for you?
